app/assets/stylesheets/spotlight/_curation.scss

```diff
@@ -149,6 +149,22 @@ h1,h2,h3,h4,h5,h6 {
   }
 }
 
+.tab-pane {
```

Comment:
Do these styles belong in https://github.com/sul-dlss/spotlight/blob/master/app/assets/stylesheets/spotlight/_bootstrap_overrides.scss?